pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -36,399 +36,142 @@ If your bot fails to start with a `PrivilegedIntentsRequired` exception, this in
 ---
 
 ### Configure the Bot
-You now have both the bot's code and a server to run it on. It's time you to connect the two by changing the bot's configurations.
+You now have both the bot's code and a server to run it on. It's time for you to connect the two by changing the bot's configurations.
+This can be done either automatically or manually, and we'll be detailing the steps for both.
+
+One thing to know is that the bot relies on precisely **two** configuration files to work
 
-#### config.yml
-Entering the directory of the cloned code, you will find a file named `config-default.yml`.
+#### .env.server
+All server configuration values are saved in this file, which needs to be at the root directory of the cloned code.
 This file contains the various configurations we use to make the bot run on the Python Discord server, such as channel and role IDs, and the emojis it works with.
 It also contains configurations such as how long it takes for a help channel to time out, and how many messages a user needs to voice-verify.
 
-To run the bot in your test server, you will need to override some of those configurations.
-Create and open a new file in the directory called `config.yml`. Alternatively, copy the `config-default.yml` file and rename the copy to `config.yml`.
-The bot will first look at the items in `config.yml`, and will fall back to `config-default.yml` only if necessary. Note that you don't have to specify all items in `config.yml`, just the ones you want to override such as channel IDs.
+This file will be created for you automatically if you decide to go with [automatic configuration](#automatic-configuration),
+otherwise a lot of it has to be done by hand which will be detailed in the [manual configuration](#manual-configuration) section.
 
-See [here](../obtaining-discord-ids) for help with obtaining Discord IDs.
+#### .env
+This file will mostly contain sensitive information such as your `BOT_TOKEN` and your `REDIS_PASSWORD`.
+It will also contain configurations related to external services the bot might use such as `USE_METRICITY`, which are unrelated your server, with the only exception being `GUILD_ID`.
```

Comment:
```suggestion
It will also contain configurations related to external services the bot might use such as `USE_METRICITY`, which are unrelated to your server, with the only exception being `GUILD_ID`.
```
This one's on me.